openfin/index.d.ts

```diff
@@ -63,7 +63,7 @@ declare namespace fin {
 		 */
 		close(force?: boolean, callback?: () => void, errorCallback?: (reason: string) => void): void;
 		/**
-		 * Retrieves an array of wrapped fin.desktop.Windows for each of the application�s child windows.
+		 * Retrieves an array of wrapped fin.desktop.Windows for each of the application�s child windows.
```

Comment:
Could you change these to regular apostrophes or just revert the line please?

A few more occurrences below